gatsby-config.js

```diff
@@ -170,7 +172,7 @@ const plugins = [
               }
             }
           }
-        `
+    `
     },
     resolve: `gatsby-plugin-feed`
```

Comment:
not related directly to this PR ask: let's unify the order - plugin name -> options if possible

Reply:
I'm totally for that, just didn't want to go out of scope without a reason. This shouldn't take long.